src/engine/compiler/MacroAssembler.v3

```diff
@@ -290,6 +290,8 @@ class MacroAssembler(valuerep: Tagging, regConfig: RegConfig) {
 	def emit_call_runtime_callHost(func_arg: Reg);
 	def emit_jump_to_trap_at(reason: TrapReason);
 	def emit_call_runtime_op(op: Opcode);
+	def emit_get_curstack(r: Reg);
```

Comment:
I recently started refactoring `MacroAssembler` to have explicit methods per field access, e.g. `emit_v3_Array_elem_r_rr` and so on. I think this will be cleaner if we also follow that convention here.

Those methods are now used exclusively in the JIT, and I think we should refactor the interpreter to follow suit.